src/Extracting/Strategies/UrlParameters/GetFromLaravelAPI.php

```diff
@@ -69,11 +69,15 @@ public function __invoke(ExtractedEndpointData $endpointData, array $routeRules)
                     $type = $this->normalizeTypeName($typeName);
                     $parameters[$paramName]['type'] = $type;
 
-                    // If the user explicitly set a `where()` constraint, use that to refine examples
-                    $parameterRegex = $endpointData->route->wheres[$paramName] ?? null;
-                    $example = $parameterRegex
-                        ? $this->castToType($this->getFaker()->regexify($parameterRegex), $type)
-                        : $this->generateDummyValue($type);
+                    $example  = optional($argumentInstance::first())->id;
```

Comment:
Use `??`, not `optional()`.

Reply:
One more thing: please wrap this in a try/catch, so it doesn't crash if the database isn't available.